set.mm

```diff
@@ -30533,22 +30537,24 @@ Such interpretation is rarely needed (see also ~ df-ral ).  (Contributed
     $( Equality theorem for restricted class abstractions.  Inference form of
        ~ rabeqf .  (Contributed by Glauco Siliprandi, 26-Jun-2021.) $)
     rabeqif $p |- { x e. A | ph } = { x e. B | ph } $=
-      ( wceq crab rabeqf ax-mp ) CDHABCIABDIHGABCDEFJK $.
+      ( wceq crab cv wcel wa cab nfth eleq2 anbi1d abbid df-rab 3eqtr4g ax-mp )
+      CDHZABCIZABDIZHGUABJZCKZALZBMUDDKZALZBMUBUCUAUFUHBUABGNUAUEUGACDUDOPQABCR
+      ABDRST $.
   $}
 
   ${
     $d x A $.  $d x B $.
     $( Equality theorem for restricted class abstractions.  (Contributed by NM,
        15-Oct-2003.) $)
     rabeq $p |- ( A = B -> { x e. A | ph } = { x e. B | ph } ) $=
-      ( nfcv rabeqf ) ABCDBCEBDEF $.
+      ( wceq cv wcel eleq2 anbi1d rabbidva2 ) CDEZAABCDKBFZCGLDGACDLHIJ $.
 
     ${
       rabeqi.1 $e |- A = B $.
```

Comment:
This is identical to rabeqif if the two nf-hypotheses are dropped.  You don't need the two dv conditions either.  Drop this theorem.

Reply:
The same would hold for rabeqi then (which would be identical with rabeqif which could also be dropped then).